writings/references.md

```diff
@@ -0,0 +1,11 @@
+# References
+
+Listed in rough order of most to least relevant.
+
```

Comment:
Yay. I've wanted the reading list to migrate off the readme for a while.  At some point we should have a sort of living wiki.  But this is a nice cleanup.  Nice phrasing.   

Might be good to use numbered items instead of bullets so that paper refs are less positional sensitive in the sentence placement? But lookin good